x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/inference/trainedmodel/RobertaTokenization.java

```diff
@@ -51,7 +51,7 @@ public static RobertaTokenization fromXContent(XContentParser parser, boolean le
 
     private final boolean addPrefixSpace;
 
-    private RobertaTokenization(
+    public RobertaTokenization(
```

Comment:
Why is there a constructor that doesn't expect `doLowerCase` for `RobertaTokenization`? The only public constructor before this change forces lower case to `false`. This seems something worth clarifying as we're refactoring the area.

Reply:
I reverted this change as it was only a convenience for a test. The value of `doLowerCase` should always be false.